### PR TITLE
:lipstick: Reduce main title size

### PR DESCRIPTION
**Pull request by PuneetDabral**
## Related Issue 
 reduce font size of the main title

Closes: #36 

### Describe the changes you've made

I replace `text-5xl` Tailwind class to `text-3xl`

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
Screenshot (101) | Screenshot (100) |

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,7 +81,9 @@
 
         <!--Left Col-->
         <div class="flex flex-col w-full md:w-2/5 justify-center items-start text-center md:text-left">
-          <h1 class="my-4 text-5xl font-bold leading-tight">
+
+          <!-- <p class="uppercase tracking-loose w-full">What business are you?</p> -->
```
Comment (mkubdev) [inline index.html:85]: Hi, you should remove this line, we cleaned it up several commits ago :)

**Review by mkubdev — CHANGES_REQUESTED** [review]
Thanks for your pull request @PuneetDabral :)

* Can you update your PR title with (for example) : "reduction of the font size of the main title" ?
* Can you please update the PR content and add screenshot ? (One from the current size, and one from your modification) 
* Can you please remove the commented line ? Just make another commit and push it.

🌞

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see comments by @mkubdev.

**Review by Panquesito7 — APPROVED** [review]
Looks much better! Thank you for your contribution! 🚀

**Comment by mkubdev** [comment]
LGTM! Thank you for your contribution! 🚀

**Review by mkubdev — APPROVED** [review]
LGTM! Thank you for your contribution! 🚀

**Pull request merged**